modules/pensions/lib/pensions/monitor.rb

```diff
@@ -237,13 +260,16 @@ def track_submission_exhaustion(msg, claim = nil)
     # @param e [Exception]
     #
     def track_send_confirmation_email_failure(claim, lighthouse_service, user_uuid, e)
-      Rails.logger.warn('Lighthouse::PensionBenefitIntakeJob send_confirmation_email failed', {
-                          claim_id: claim&.id,
-                          benefits_intake_uuid: lighthouse_service&.uuid,
-                          confirmation_number: claim&.confirmation_number,
-                          user_uuid:,
-                          message: e&.message
-                        })
+      additional_context = {
+        confirmation_number: claim&.confirmation_number,
+        user_uuid:,
+        claim_id: claim&.id,
+        benefits_intake_uuid: lighthouse_service&.uuid,
+        message: e&.message
+      }
+      log_silent_failure(additional_context, user_uuid, call_location: caller_locations.first)
```

Comment:
i think we are holding off on alerting  when sending an email fails, for now (??), since it is not an error/issue with the claim submission

Reply:
@wayne-weibel I didn't add this function. It was already there

Reply:
k this line can be removed